srv/salt/ceph/openattic/default.sls

```diff
@@ -5,3 +5,13 @@ install openattic:
     - pkgs:
       - openattic
 
+enable openattic-rpcd:
```

Comment:
I had the same thought. Salt just makes sure that the service is started...so no harm done I figured.